samza-kv/src/main/scala/org/apache/samza/storage/kv/KeyValueStorageEngineMetrics.scala

```diff
@@ -34,8 +34,8 @@ class KeyValueStorageEngineMetrics(
   val deletes = newCounter("deletes")
   val flushes = newCounter("flushes")
 
-  val restoredMessages = newCounter("messages-restored")
-  val restoredBytes = newCounter("messages-bytes")
+  val restoredMessages = newGauge("messages-restored", 0)
```

Comment:
Counters are not `reported` as deltas from samza  [`JmxReporter`, `MetricsSnapshotReporter`, `AmfReporter`]. I think problem is LI metrics renderer engine `plots` the delta from the counter(diff between two different reports). It wasn't obvious from JIRA and code(replacing counter functionality with gauge). It's clear now. Thanks.